package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "dependencies": {
     "@braid/vue-formulate": "^2.5.2",
-    "@chatwoot/prosemirror-schema": "1.0.3",
+    "@chatwoot/prosemirror-schema": "1.0.4",
```

Comment:
@nithindavid The version tag is not published on NPM.
<img width="797" alt="image" src="https://github.com/chatwoot/chatwoot/assets/64252451/374b66fd-819e-4cac-afad-b3c68aaf62a3">